source/D2Common/src/Units/Units.cpp

```diff
@@ -1501,10 +1494,7 @@ __forceinline void __fastcall UNITS_UpdateRunWalkAnimRateAndVelocity(D2UnitStrc*
 
 		if (nUnitType == UNIT_PLAYER)
 		{
-			pCharStatsTxtRecord = UNITS_GetCharStatsTxtRecord(nClassId);
-			*(uint32_t*)v60 = 0;
-			*(uint16_t*)&v60[1] = pCharStatsTxtRecord->nWalkSpeed;
```

Comment:
In the case right here, casting to a uint16 and writing to [1] sets the 3rd byte to the velocity: 6 -> 0x00060000. Stepping through the vanilla game, it shifts 8 instead just like monsters: 6 -> 0x00000600. Would make you run much faster than intended haha!